spartan/aztec-network/templates/reth.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if not .Values.network.disableEthNode }}
```

Comment:
instead of adding a new flag, could this be replaced by checking if .Values.ethereum.externalHost is set? If we wanted a flag, we could create a helper based off checking that value?